js/script.js

```diff
@@ -117,7 +122,17 @@ const createTaskElement = (task) => {
 
     $taskElement.id = task.id;
 
-    $taskDetails.innerHTML = `${task.title}, ${task.createdAt} `;
+    if (task.isDone) {
+        const $strikeTaskDetails = document.createElement("s");
+        $strikeTaskDetails.innerHTML = `${task.title}, ${task.createdAt} `;
+
+        $taskDetails.appendChild($strikeTaskDetails);
+
+        $editButton.hidden = true;
+        $doneButton.hidden = true;
+    } else {
+        $taskDetails.innerHTML = `${task.title}, ${task.createdAt} `;
+    }
```

Comment:
Please move it to it's own branch. 

Reply:
@mza-wtag bhai this changes are for this branch because i am re rendering the task list only during search. I am rendering the list again when a todo is marked done.